src/Mod/AddonManager/addonmanager_firstrun.py

```diff
@@ -51,55 +52,28 @@ def exec(self) -> bool:
             warning_dialog = FreeCADGui.PySideUic.loadUi(
                 os.path.join(os.path.dirname(__file__), "first_run.ui")
             )
-            autocheck = self.pref.GetBool("AutoCheck", False)
-            download_macros = self.pref.GetBool("DownloadMacros", False)
-            proxy_string = self.pref.GetString("ProxyUrl", "")
-            if self.pref.GetBool("NoProxyCheck", True):
-                proxy_option = 0
-            elif self.pref.GetBool("SystemProxyCheck", False):
-                proxy_option = 1
-            elif self.pref.GetBool("UserProxyCheck", False):
-                proxy_option = 2
-
-            def toggle_proxy_list(option: int):
-                if option == 2:
-                    warning_dialog.lineEditProxy.show()
-                else:
-                    warning_dialog.lineEditProxy.hide()
-
-            warning_dialog.checkBoxAutoCheck.setChecked(autocheck)
-            warning_dialog.checkBoxDownloadMacroMetadata.setChecked(download_macros)
-            warning_dialog.comboBoxProxy.setCurrentIndex(proxy_option)
-            toggle_proxy_list(proxy_option)
-            if proxy_option == 2:
-                warning_dialog.lineEditProxy.setText(proxy_string)
-
-            warning_dialog.comboBoxProxy.currentIndexChanged.connect(toggle_proxy_list)
-
-            warning_dialog.labelWarning.setStyleSheet(
-                f"color:{utils.warning_color_string()};font-weight:bold;"
-            )
 
+            # Set up the warning icon
+            warning_icon_label = warning_dialog.warningIconLabel
+            pixmap = QPixmap(":/icons/AddonMgrWithWarning.svg")
+            warning_icon_label.setPixmap(pixmap)
+            warning_icon_label.setScaledContents(True)
+            warning_icon_label.setFixedSize(80, 80)
+
+            # Customize layout margins and spacing
+            layout = warning_dialog.findChild(QtWidgets.QHBoxLayout, "bodyLayout")
+            layout.setContentsMargins(20, 6, 20, 6)  # Left, top, right, bottom margins
+            layout.setSpacing(30)  # Set spacing between widgets
+
```

Comment:
Can these adjustments be changed in the ui file instead?

Reply:
Thanks for feedback, I'll see what can be moved over.